packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts

```diff
@@ -116,6 +116,15 @@ export const pageViewTrackingPlugin: CreatePageViewTrackingPlugin = (options: Op
           ...pageViewEvent.event_properties,
         };
       }
+
+      // Update the pageCounter for the page view event
+      if (localConfig && (event.event_type === options.eventType || event.event_type === defaultPageViewEvent)) {
```

Comment:
Not sure we should continue to support the defaultPageViewEvent is it is overridden in Options. Could be better to set the Options.eventType = defaultEventType by default and then only use `options.eventType` in the plugin logic

```suggestion
      if (localConfig && event.event_type === options.eventType) {
```

Reply:
+1.

Reply:
Give the default event  type at the beginning of the plugin